recipes/r-twang/meta.yaml

```
@@ -0,0 +1,95 @@
+{% set version = '2.5' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-twang
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/twang_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/twang/twang_{{ version }}.tar.gz
+  sha256: fc355527c57e4f6e0f60d26d7c690c4475fcd5fb165d125fea7cc6b9fafc4ce5
+
+build:
+  merge_build_host: True           # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+  missing_dso_whitelist:
+    - '*/R.dll'                    # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}          # [not win]
+    - {{ compiler('m2w64_c') }}    # [win]
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}make
+    - {{ posix }}sed               # [win]
+    - {{ posix }}coreutils         # [win]
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+    - r-matrixmodels
+    - r-data.table
+    - r-gbm >=1.5_3
+    - r-ggplot2
+    - r-lattice
+    - r-latticeextra
+    - r-survey
+    - r-xgboost
+    - r-xtable
```

Makevars shows they link against BLAS/LAPACK. Even though it didn't show an issue, it won't hurt to have this in the metadata.

```suggestion
    - r-xtable
    - libblas
    - liblapack
```